dataprofiler/tests/profilers/test_unstructured_labeler_profile.py

```diff
@@ -131,7 +131,36 @@ def test_profile(self, processor_class_mock, model_class_mock):
 
         # key and value populated correctly
         self.assertDictEqual(expected_profile, profile)
+
+    @mock.patch('dataprofiler.profilers.'
+                'unstructured_labeler_profile.DataLabeler')
+    def test_unstructured_labeler_profile_add(self, model_class_mock):
```

Comment:
I would suggest that each profile is setup where an entity exists in both and the cases where it exists in one but not the other. This can be done by manually setting entity counts and the other parameters. Otherwise we arent truly testing all cases.

Reply:
We would not need to set a labeler mock values because we wouldn't need to update the profile, just merge.